etc/zfsbootmenu/recovery.conf.d/recovery.conf

```diff
@@ -0,0 +1,7 @@
+zfsbootmenu_teardown+=" /zbm/contrib/xhci-teardown.sh "
+install_optional_items+=" /etc/zbm-commit-hash /bin/gdisk /bin/parted /bin/mkfs.vfat /bin/mkfs.ext4 /bin/curl /bin/efibootmgr"
```

Comment:
I'm not really sure I like this, as it's very Void-specific. I've debated using a custom variable and looping through it in `module-setup.sh`. It could support things with an absolute path, or things expected to be in `$PATH` and install them as needed.

Reply:
Aren't you building this in a Void container?

Reply:
> Aren't you building this in a Void container?

Ahh yeah - that is true. It's probably fine in that case.